resources/home/dnanexus/make_workbook.py

```
@@ -829,22 +843,25 @@ def create_additional_analysis_page(self):
-
-            # Grab all the exomiser variants
-            ex_df = ex_df.loc[ex_df['Priority'] != "De novo"]
-
-            # Now we have filtered out all variants that are in the GEL tiering
-            # page we need to get the top 3 ranks in the exomiser df
-            if not ex_df.empty:
-                ex_df = var_info.get_top_3_ranked(ex_df)
-
-            # Concat de novos and exomiser ranked back together
-            ex_df = pd.concat([ex_df, denovo_df])
+            if not exomiser_df.empty:
```

Ruff (0.14.1)

Undefined name `exomiser_df`

(F821)